src/components/inputs/InlineField.tsx

```diff
@@ -69,16 +69,14 @@ export default function InlineField({
     }
   }, []);
 
-  const valueRef = useRef(value);
   const style = useMemo(
     () => ({
       ...textStyle,
-      lineHeight: android ? textStyle.lineHeight : undefined,
       marginBottom: 0,
       marginTop: 0,
-      minHeight: inputHeight + paddingVertical * 2 + (android ? 2 : 0),
-      paddingBottom: inputProps?.multiline ? (ios ? 15 : 7) : 0,
-      paddingTop: inputProps?.multiline ? (android ? 11 : 15) : android ? (valueRef.current ? 16 : 11) : 0,
+      minHeight: inputHeight + paddingVertical * 2,
+      paddingBottom: inputProps?.multiline ? (ios ? 15 : 10) : 0,
```

Comment:
Android still needs a little bit less padding, or the field becomes bigger than the others when only 1 line.